### DISCUSSION TITLE
Identidades Secretas

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#426] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/identidades-secretas/

**Comment by maryon-telvanni** [comment]
Alguns eu nem sei quem são, mas nenhum faz tanto sentido quanto esse:



![null](<https://www.vish.blog.br/imagens/identidades-secretas/images/gallery/109.webp>)

**Comment by brunoaugusto** [comment]
Eu meio que fiquei ofendido pelo artista considerar que o Link seria o Peter Pan

**Comment by najarin-the-loremaster** [comment]
> [Maryon Telvanni](</perfil/?user=maryon-telvanni>) disse [terça-feira, 31 de maio de 2022](<https://www.vish.blog.br/imagens/identidades-secretas/#comment_1142105203>):

> Alguns eu nem sei quem são, mas nenhum faz tanto sentido quanto esse:[![null](<https://camo.githubusercontent.com/9bf0a4a03a8a751224a337491ac13523121a5ba28eb2c9f368a98738c563d831/68747470733a2f2f7777772e766973682e626c6f672e62722f696d6167656e732f6964656e746964616465732d73656372657461732f696d616765732f67616c6c6572792f3130392e77656270>)](<https://camo.githubusercontent.com/9bf0a4a03a8a751224a337491ac13523121a5ba28eb2c9f368a98738c563d831/68747470733a2f2f7777772e766973682e626c6f672e62722f696d6167656e732f6964656e746964616465732d73656372657461732f696d616765732f67616c6c6572792f3130392e77656270>)

Verdade e até existem duas teorias:



Em uma delas, Samurai Jack se passa num futuro onde Aku teria derrotado as Meninas e destruído Townsville e o "Jack" seria o Professor Utônio buscando vingança; em outra, o Elemento X seria uma substância mutagênica presente na natureza da qual tanto o Aku quanto suas Sete Filhas se originaram, vivendo em confronto eterno com "Jack" que tem de se adaptar ao longo dos anos e assume a personalidade de cientista, disfarce perfeito para criar as Meninas como seu legado.
